src/Spectre.IO/Internal/File.cs

```diff
@@ -87,6 +89,9 @@ public void CreateSymbolicLink(FilePath destination)
                 throw new InvalidOperationException("Detination path cannot be relative");
             }
 
+#if NET6_0_OR_GREATER
+            System.IO.File.CreateSymbolicLink(destination.FullPath, Path.FullPath);
```

Comment:
https://source.dot.net/#System.Private.CoreLib/File.cs,725

<param name="path">The path where the symbolic link should be created.</param>
<param name="pathToTarget">The path of the target to which the symbolic link points.</param>

Reply:
Potentially we could also call
```csharp
FileSystem.CreateSymbolicLink(path, pathToTarget, isDirectory: false);
```
Directly instead of `System.IO.File.CreateSymbolicLink`